multi-agent/tests/integration/endpoints/test_builtins_endpoints.py

```
@@ -0,0 +1,389 @@
+        resp = client.put(
+            "/api/resources/builtin.update",
+            json={"resourceId": "r1", "name": "renamed"},
+            headers=admin_headers,
+        )
+
+        assert resp.status_code == 409
+        builtin_resource_service.update_builtin_with_cascade.assert_not_called()
+
+
+class TestToggleBuiltinVisibility:
+    def test_requires_admin(self, client, user_headers, builtin_resource_service):
```

Ruff (0.16.1)

[warning] Unused method argument: `builtin_resource_service`

(ARG002)